### ISSUE TITLE
Navigation events stop firing after modal is opened

### ISSUE THREAD

**Issue by beetlebum** [software-mansion/react-native-screens#278]
I have a bottom tab navigator (React Navigation 4.0.10, React Native 0.62-rc0, React Native Screens 2.0 alpha23) and one of the tabs is listening for navigation events.
All stack navigators are setup using `createNativeStackNavigator`.

When the application opens I can see navigation events being triggered when the tab is selected:
```
 LOG  willBlur
 LOG  didBlur
 LOG  willFocus
 LOG  didFocus
 LOG  willBlur
 LOG  didBlur
```

If a modal opens (`mode: 'modal',`) and closes, these events above do not get triggered anymore.

If I switch to `react-navigation-stack` the error doesn't happen.

Any ideas?

**Comment by mironiasty** [comment]
It should be fixed with in recent release [2.0.0-beta.3](https://github.com/software-mansion/react-native-screens/releases/tag/2.0.0-beta.3)

**Comment by kmagiera** [comment]
Closing this one as it's been resolved in 2.0-beta.3. In case it does not work for you @beetlebum please comment here or start a new issue.
